### ISSUE TITLE
Interpreter not installing

### ISSUE THREAD

**Issue by brettstark73** [KillianLucas/open-interpreter#623]
### Describe the bug

Can not install via pip install

pip install open-interpreter

### Reproduce

pip install open-interpreter
ERROR: Could not find a version that satisfies the requirement open-interpreter
ERROR: No matching distribution found for open-interpreter

### Expected behavior

download and install

### Open Interpreter version

na

### Python version

Python 3.9.1

### Operating System name and version

Mac OS 14.0

**Comment by ericrallen** [comment]
Hey there, @brettstark73!

Open Interpreter currently [supports Python 3.10](https://github.com/KillianLucas/open-interpreter/blob/main/pyproject.toml#L12) and 3.11. I think some of the underlying dependencies still need to update to support 3.12.

There was some talk of supporting down to 3.8, but I’m not sure what the status of that initiative is.

So, for the moment, you’ll need to install a compatible version of Python to use Open Interpreter.

**Comment by brettstark73** [comment]
yes, working now thanks. once i finally figured out how to get pyenv to work :(.
